test/drop_test/event.py

```diff
@@ -417,3 +417,37 @@ def __eq__(self, rhs):
 
     def __str__(self):
         return f"RuntimeError(status={self._status})"
+
+
+class TransferDeferred(Event):
+    def __init__(
+        self,
+        uuid_slot: int,
+        peer: str,
+        status: int,
+        os_err: typing.Optional[int] = None,
+        ignore_os: bool = False,
```

Comment:
I am not exactly sure what `ignore_os` does, could you explain it? It's not used in the only testcase we have

Reply:
It's the same as with `TransferFailed` event. It means that when we compare events we ignore if they differ in the OS error code